po/ru.po

```diff
@@ -0,0 +1,182 @@
+# Russian translations for app.drey.EarTag package
+# Английские переводы для пакета app.drey.EarTag.
+# Copyright (C) 2022 THE app.drey.EarTag'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the app.drey.EarTag package.
+#  Igor Dyatlov <dyatlov.igor@protonmail.com>, 2022.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: app.drey.EarTag\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-07-26 21:51+0300\n"
+"PO-Revision-Date: 2022-08-05 19:34+0300\n"
+"Last-Translator: Automatically generated\n"
+"Language-Team: Igor Dyatlov <dyatlov.igor@protonmail.com>\n"
+"Language: ru\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 "
+"|| n%100>=20) ? 1 : 2);\n"
+"X-Generator: Poedit 3.1\n"
+
+#: data/app.drey.EarTag.desktop.in:3 data/app.drey.EarTag.appdata.xml.in:3
+msgid "Ear Tag"
+msgstr "Ушная бирка"
```

Comment:
II don't think that the name should be translated; despite being named after a real-life thing I treat it more like an original name. I'll probably update the files to include a note for translators about this.

(I should probably add a "generic name" field in the .desktop file as a more translation-friendly alternative; for now the description sort of covers that.)

Reply:
I returned original name and added a comment it to the template for translators